DirectX11/HackerDXGI.cpp

```diff
@@ -1947,6 +2148,26 @@ STDMETHODIMP HackerDXGISwapChain::GetDesc(THIS_
 	HRESULT hr = mOrigSwapChain->GetDesc(pDesc);
 	if (hr == S_OK)
 	{
+		if (pDesc)
+		{
+			//TODO: not sure whether the upscaled resolution or game resolution should be returned
```

Comment:
I'd follow the lead of the Direct Mode code, and return the forced resolution.  I'm also not sure if that is correct.

The problem is that our runtime environment is unbelievably complex, and includes crazy stuff like other injectors (ENB, Reshade), Steam overlay, Origin overlay, MSI Afterburner, and on and on.  

If we return the original resolution when one of those asks for the description, they'd get the wrong size, and draw their overlay wrong. 

No idea if that actually happens or not, just observing some of the edge cases we can run into.  We can revisit it if it turns out to be a problem.